.github/ISSUE_TEMPLATE/team_recommend.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Team recommendation
+about: Nominate yourself or another Organization Member to join a team
+title: ''
+labels: nomination
+assignees: ''
+
+---
+
+*For details on how this process works, please see our [contributor roles documentation](https://github.com/instructlab/community/blob/main/CONTRIBUTOR_ROLES.md)*
+
+**Which member are you nominating for a team change?**
+<!-- The GitHub username of the Organization Member you are nominating -->
+
+**Which team are you nominating for this member for?**
+<!-- For SDG, the only available team is SDG Maintainers. If you'd like to nominate for a different team, please refer to the contributor roles documentation linked above. -->
```

Comment:
Per the org governance, this is incorrect - everyone has a Triager level team: https://github.com/instructlab/community/blob/main/CONTRIBUTOR_ROLES.md#current-roles

Reply:
I didn't see any traiger level team for SDG -- https://github.com/orgs/instructlab/teams
Please lmk if I am missing something here @nathan-weinberg

Reply:
Just because the team doesn't exist doesn't mean it shouldn't - don't know why it doesn't, but either way the text is still incorrect. If you wish to submit a PR to the overall org governance to change how the contributor roles work that is fine, but otherwise please align the text to community doc.

Reply:
Fair, updated the description.